### PR TITLE
Feature/dev 65

### PR DESCRIPTION
**Pull request by MarcinSkic**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -89,6 +89,62 @@ export const selectableAlbumSchema = getAlbumSchema(trackSchema).and(z.object({
...
+export const createdGameDtoSchema = z.object({
+  gameId: z.string().length(4),
+  hostGuid: z.string().uuid(),
+})
+export type CreatedGameDto = z.infer<typeof createdGameDtoSchema>
+
+const messageTypeString = 'message'
+const errorTypeString = 'messageError'
+
+export const messageSchema = z.discriminatedUnion('$type', [
```
Comment (FilipTarajko) [inline src/types.ts:104]: why

File: src/components/setup/HostView.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+async function handleGameStart() {
+  if (!musicPlayerStore.ready)
+    return
+
+  await musicPlayerStore.turnOn()
+  const tracks = await spotifyLibraryStore.getTracksFromSelectedSets(access_token, router)
+  gameDataStore.prepareGame(tracks)
+  router.push({ name: 'round', params: { id: '7734' } })
```
Comment (FilipTarajko) [inline src/components/setup/HostView.vue:31]: i love magic number strings in pull requests

Comment (MarcinSkic) [reply src/components/setup/HostView.vue:31]: Will be deleted when next task is done, not relevant

File: src/types.ts (hunk 1 above)
Comment (MarcinSkic) [reply src/types.ts:104]: learn to ask better questions

**Pull request merged**